hexrd/fitting/fitpeak.py

```diff
@@ -58,15 +56,15 @@ def cnst_fit_obj(x, b):
     return np.ones_like(x)*b
 
 
-def cnst_fit_jac(x, b):
+def cnst_fit_jac(x, _b):
     return np.vstack([np.ones_like(x)]).T
 
 
 def lin_fit_obj(x, m, b):
     return m*np.asarray(x) + b
 
 
-def lin_fit_jac(x, m, b):
+def lin_fit_jac(x, _m, _b):
```

Comment:
Same comment as prior - if `_m` and `_b` aren't being used by the function, we shouldn't have them.

Reply:
Same as above